src/composable/orderTypes/Twap.ts

```diff
@@ -350,11 +350,37 @@ export class Twap extends ConditionalOrder<TwapData, TwapStruct> {
     const { numberOfParts } = this.data
     const startTimestamp = await this.startTimestamp(params)
 
+    if (blockTimestamp < startTimestamp) {
+      return {
+        result: PollResultCode.UNEXPECTED_ERROR,
```

Comment:
Shouldn't this return the error indicating to try again at `x` block instead of unnexpected error?

Reply:
Good question, but in my opinion it shouldn't. That's the responsibility of the `pollValidate` which should have returned the error you say, so no-one should have called this function.

Additionally, this function should be called after getting the order data, which would also fail for the same reason. 

Therefore, this `UNEXPECTED_ERROR` is just signaling that something unexpected happened, so it can be matched in DEV time.

Reply:
Given that this is an execution path that shouldn't occur (and thus as mentioned, is actually an _unexpected error_, perhaps a little note in the code for maintainers along the lines of "This shouldn't happen as `pollValidate` should have caught before attempting a poll", or something similar.